app/enterprise/2.1.x/plugins/oidc-okta.md

```diff
@@ -56,6 +56,14 @@ as an authorized redirect URI in Okta (under the **Authentication** section of y
 
     <img src="https://doc-assets.konghq.com/0.35/plugins/oidc-okta/07-auth-server-settings.png">
 
+7. If you created a new Authorization server in step 5, also create a default Access Policy.
+
+    <img src="https://doc-assets.konghq.com/0.35/plugins/oidc-okta/not-yet-exists.png">
```

Comment:
The new process and syntax for referencing screenshots is documented here: https://konghq.atlassian.net/wiki/spaces/KD/pages/1329955371/Adding+screenshots+to+docs. We no longer upload images to AWS https://konghq.atlassian.net/wiki/spaces/KD/pages/1027964929/OUTDATED+DO+NOT+USE+Adding+Screenshots+to+Docs; the images should reside in the docs repo. Also, the images should be cleanly cropped and a border applied. The Add Claim screenshot is actually an Edit Claim, so that screenshot should be retaken.

![image](https://user-images.githubusercontent.com/3756245/98962950-c14e6200-24cc-11eb-9a7e-f43072e326da.png)

Reply:
![image](https://user-images.githubusercontent.com/3756245/98963385-2b670700-24cd-11eb-88bb-5a2dd5fc4396.png)

Direct review link: https://deploy-preview-2450--kongdocs.netlify.app/enterprise/2.1.x/plugins/oidc-okta/